pcweb/components_webpage/navbar/navbar.py

```diff
@@ -0,0 +1,204 @@
+                    padding_bottom="20px",
+                    spacing="2",
+                ),
+                rx.flex(
+                    resource_header("Resources"),
+                    resources_item(
+                        "Component Library", "/docs/library", "layout-panel-left"
+                    ),
+                    resources_item(
+                        "Roadmap",
+                        "https://reflex-dev.notion.site/d1b60926ced14914bdbb42547ac84e50?v=723e0afc57294e40961d177aa691ee37",
+                        "map-pinned",
+                    ),
+                    resources_item("Blog", "/blog", "text"),
+                    resources_item("FAQ", "/faq", "list-todo"),
+                    direction="column",
+                    align_items="start",
+                    padding_top="20px",
+                    padding_bottom="20px",
+                    spacing="2",
+                ),
+                rx.flex(
+                    resource_header("Hosting"),
+                    resources_item(
+                        "Deployment", "/docs/hosting/deploy-quick-start/", "globe"
+                    ),
+                    resources_item(
+                        "Self-Hosting", "/docs/hosting/self-hosting/", "server"
+                    ),
+                    direction="column",
+                    background="linear-gradient(218deg, #1D1B23 -35.66%, #131217 100.84%);",
+                    border_left="1px solid rgba(29, 29, 32, 0.08);",
+                    align_items="start",
+                    height="210px",
+                    padding_top="20px",
+                    padding_left="20px",
+                    padding_bottom="20px",
+                    padding_right="20px",
+                    spacing="2",
+                ),
+                spacing="6",
+            ),
+            border="1px solid rgba(29, 29, 32, 0.08);",
+            background="linear-gradient(180deg, rgba(29, 27, 33, 0.95) 0%, rgba(20, 19, 24, 0.95) 100%);",
+            box_shadow="0px 24px 54px -17px rgba(13, 12, 16, 0.30), 0px 0px 0px 1px rgba(93, 93, 107, 0.29), 0px 0px 64px 5px rgba(53, 51, 60, 0.30) inset;",
+            max_width="1000px",
+            height="210px",
+            padding="0",
+            overflow="hidden",
+        ),
+    )
+
+
+def navigation_section():
+
+    section_style={
+        "color": "#6C6C81",
+        "font-weight": "400",
+    }
+
+    return rx.box(
+        rx.flex(
+            rx.link(rx.text("Docs", href="/docs/getting-started/introduction/", style=section_style)),
+            rx.link(rx.text("Blog", href="/blog", style=section_style)),
+            rx.link(rx.text("Gallery", href="/docs/gallery", style=section_style)),
+            resources_section(style=section_style),
+            spacing="5",
+        ),
+        display=["none", "none", "none", "none", "flex", "flex"],
+    )
+
+def blur_background():
+    return rx.fragment(
+        rx.script(
```

Comment:
nice! You could also try to use masen's intersection observer library here https://github.com/masenf/reflex-intersection-observer